app/src/main/java/com/teamwss/websoso/ui/onboarding/third/OnboardingThirdFragment.kt

```diff
@@ -2,17 +2,42 @@ package com.teamwss.websoso.ui.onboarding.third
 
 import android.os.Bundle
 import android.view.View
+import androidx.fragment.app.activityViewModels
 import com.teamwss.websoso.R
 import com.teamwss.websoso.databinding.FragmentOnboardingThirdBinding
 import com.teamwss.websoso.ui.common.base.BindingFragment
+import com.teamwss.websoso.ui.onboarding.OnboardingViewModel
+import com.teamwss.websoso.ui.onboarding.third.adapter.GenreAdapter
+import com.teamwss.websoso.ui.onboarding.third.model.Genre
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class OnboardingThirdFragment :
     BindingFragment<FragmentOnboardingThirdBinding>(R.layout.fragment_onboarding_third) {
+    private val onboardingViewModel: OnboardingViewModel by activityViewModels()
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        setupGenreAdapter()
+        setupObserver()
 
+        binding.viewModel = onboardingViewModel
+        binding.lifecycleOwner = viewLifecycleOwner
     }
-}
\ No newline at end of file
+
+    private fun setupGenreAdapter() {
+        val genres = Genre.values().toList()
```

Comment:
c: 
```suggestion
        val genres = Genre.entries.toList()
```

엔트리 써먹기(나도 리뷰받앗던 거 후후)